include/ipmitool/ipmi_nm.h

```diff
@@ -169,3 +173,13 @@ struct nm_suspend {
 } __attribute__ ((packed));
 
 int ipmi_nm_main(struct ipmi_intf *intf, int argc, char **argv);
+
+static inline void nm_set_id(void *buf)
+{
+	htoipmi24(IPMI_NM_ID, buf);
+}
+
+static inline bool nm_check_id(void *buf)
+{
+	return IPMI_NM_ID != ipmi24toh(buf);
```

Comment:
this should be:
	return IPMI_NM_ID == ipmi24toh(buf);

Because when it is called, it bails if this function returns false.

Reply:
Great catch! Will fix. Thanks.